cEP-0010.md

```diff
@@ -136,10 +138,34 @@ Here is the list of aspects to be implemented along with their corresponding
 
 - `Root.Formatting`: This aspect checks on the format (the structure) of your
   codebase.
-- `Root.Formatting.Quotation`: This aspect checks on the quotations being used
-  in your files given `Root.Formatting.Quotation.preferred_quotation`.
+- `Root.Formatting.Quotation`: This aspect represents the quotation marks
+  being used in the files written in languages with multiple string delimiters,
+  which is common in recent languages, and excludes C-ish languages.
 - `Root.Formatting.Quotation.preferred_quotation`: This taste represents the
-  user's preferred quotation.
+  preferred quotation marks for the selected files. It ensures that every string
+  contains the selected style of quotes.
+  For example, `'This is single quoted'` is changed to `"This is single quoted"`
+  if `preferred_quotation` is `"`.
+- `Root.Formatting.Quotation.escape_delimiter`: This taste ensures that
+  selected quotes are strictly followed even in cases where other quotation
+  marks are present inside string containing the selected style of quotes.
+  This is done by replacing other quotes with selected quotes and escaping
+  them wherever necessary.
+  For example, `"Hey 'coala'"` is changed to `"Hey \"coala\""` by setting
+  `preferred_quotation` as `"` and `escape_delimiter` as `True`.
+  Similarly, `'Hey "coala"'` is changed to `"Hey \"coala\""` with the
+  same settings as mentioned above.
+  For languages which do not support escaping quotes like `\"`, a warning
+  is shown. For such languages, taste `alternate_delimiter_language` is used
+  for quotation consistency.
+- `Root.Formatting.Quotation.alternate_delimiter_language`: This taste is
```

Comment:
Im not sure we need this. I believe `language` was moved into the core of aspects and can be used by any aspect/taste.

Reply:
So, it means any aspect/taste can access `language` so we don't need to explicitly ask for it.

I am changing this too  `alternate_delimiter`. 

Thanks for the quick response!

Reply:
So, it means any aspect/taste can access `language` so we don't need to explicitly ask for it.
Moreover, even if this taste can't access the language, the bear can. You are absolutely right on your part.

I am changing this too  `alternate_delimiter` which takes in a bool. 

Thanks for the quick response!